lte/gateway/c/oai/tasks/s1ap/s1ap_mme_itti_messaging.h

```diff
@@ -73,10 +73,10 @@ void s1ap_mme_itti_nas_non_delivery_ind(
 int s1ap_mme_itti_s1ap_path_switch_request(
     const sctp_assoc_id_t assoc_id, const uint32_t enb_id,
     const enb_ue_s1ap_id_t enb_ue_s1ap_id,
-    const e_rab_to_be_switched_in_downlink_list_t const*
+    const e_rab_to_be_switched_in_downlink_list_t* const
         e_rab_to_be_switched_dl_list,
-    const mme_ue_s1ap_id_t mme_ue_s1ap_id, const ecgi_t const* ecgi,
-    const tai_t const* tai, const uint16_t encryption_algorithm_capabilitie,
+    const mme_ue_s1ap_id_t mme_ue_s1ap_id, const ecgi_t* const ecgi,
+    const tai_t* const tai, const uint16_t encryption_algorithm_capabilitie,
```

Comment:
not related to the changes but can we fix the typo on `encryption_algorithm_capabilities`?